website/src/app/material/material.module.ts

```diff
@@ -0,0 +1,46 @@
+import { NgModule } from '@angular/core';
```

Comment:
Making one mega-module out of all possible material modules and importing it everywhere is a bad idea and makes bundles big. You should import only the ones you need in a module you need.